src/css/custom.css

```
@@ -97,4 +123,119 @@ h1 {
+}
+
+.version-selector-divider__line {
+  display: block;
+  width: 1px;
+  height: 24px;
+  background: var(--version-selector-divider-color);
+}
+
+.navbar .dropdown {
+  --version-dropdown-offset: 16px;
+  padding: 0 4px;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)